src/common/enums/sort-styles.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum SortStyles {
+    name_asc = 'name_asc',
+    name_desc = 'name_desc',
+    date_asc = 'date_asc',
+    date_desc = 'date_desc',
+}
```

Comment:
enum viet HOA hết chứ ko viết snake case, key la HOA con value co the la gi cx dc